CMakeLists.txt

```diff
@@ -21,9 +22,15 @@ endmacro (declare_mod)
 declare_mod (vcf)
 
 # Dependency libraries
+if (BUILD_STATIC)
+  set(Boost_USE_STATIC_LIBS ON) # only find static libs
+endif (BUILD_STATIC)
 find_package (Boost COMPONENTS filesystem program_options regex system REQUIRED )
 include_directories (${Boost_INCLUDE_DIR} )
 
+add_library(sqlite3 lib/sqlite/sqlite3.c)
+find_package (Threads REQUIRED)
+
 # Static build extra flags
 if (BUILD_STATIC)
```

Comment:
The file could seem more structured by moving the "Static build extra flags" before "Dependency libraries", so both `BUILD_STATIC` blocks were merged into one.